### PR TITLE
Add email and print options.

### PR DESCRIPTION
**Pull request by gcedo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -131,7 +131,7 @@
     "watch": "parallelshell 'npm run doc:watch' 'npm run prepublish:watch' 'npm run serve'"
   },
   "dependencies": {
-    "@economist/component-icon": "^3.6.6",
+    "@economist/component-icon": "5.9.0",
```
Comment (keithamus) [inline package.json]: Updating major version would be a breaking change but your commit doesn't reflect this.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@economist/component-sharebar",
-  "version": "3.0.1",
+  "version": "3.1.0",
```
Comment (keithamus) [inline package.json]: Please lets use provision and not bump & manually publish any more.

Comment (umbobabo) [inline package.json]: Sounds to me that we need a major here.

File: package.json (hunk 1 above)
Comment (gcedo) [inline package.json]: I have removed the version bump commit, sorry, I had forgotten about the new auto-publish feature.

Comment (umbobabo) [inline package.json]: If @keithamus agree we can merge this PR here and @gcedo can do another PR to provision the new devpack and then bump the major.
@keithamus pls merge if you like the plan.

Comment (keithamus) [inline package.json]: Sounds good to me, but lets get rid of the bump commit.
